app/Commands/ListRecipeCommand.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Commands;
+
+use LaravelZero\Framework\Commands\Command;
+
+class ListRecipeCommand extends Command
+{
+    protected $signature = 'list:recipes';
+
+    protected $description = 'List the available recipes';
+
+    public function handle(): void
+    {
+        $this->newLine();
+        $this->line('Mise <info>v' . config('app.version') . '</info> ');
+        $this->newLine();
+        $this->line('<fg=yellow>Recipies:</>');
+        $this->line($this->recipes());
+        $this->line('<fg=yellow>Applying recipes:</>');
+        $this->line('    mise apply [recipies]');
+        $this->newLine();
+        $this->line('<fg=yellow>Examples:</>');
+        $this->line('    mise apply tighten/basic-sass');
+        $this->line('    mise apply tighten/basic-sass laravel/local-developer-tooling');
```

Comment:
```suggestion
        $this->line('    mise apply tighten/basic-saas');
        $this->line('    mise apply tighten/basic-saas laravel/local-developer-tooling');
```

Reply:
FYI, I am very, very likely to drop the namespace for all recipes that come out of the box, and then namespaces will be for recipes that come from GitHub/Packagist